imports/server/api_keys.ts

```diff
@@ -21,7 +21,7 @@ Meteor.methods({
     check(this.userId, String);
     check(forUser, Match.Optional(String));
 
-    const user = userForKeyOperation(<string> this.userId, forUser);
+    const user = userForKeyOperation(this.userId, forUser);
```

Comment:
Oh nice -- I guess the `check()` above indicates that if `this.userId` is falsey that we diverge, so `this.userId` is always `string`?

Reply:
Yeah. I augmented the type declarations for `check` a while back so it acts as a type assertion: DefinitelyTyped/DefinitelyTyped#41414, which means that TS knows enough to narrow the type. I think this is another case where the typecast predates that change and has been unnecessary for a while.